products/spanner/api.yaml

```diff
@@ -143,3 +144,8 @@ objects:
           execute atomically with the creation of the database: if there is an
           error in any statement, the database is not created.
         input: true
+      - !ruby/object:Api::Type::String
+        name: 'state'
+        description: An explanation of the status of the database.
+        output: true
+        exclude: true
```

Comment:
Can we include this so Ansible picks it up too?

Reply:
It shouldn't be included there (or in other, later, providers) - its presence here is for historical reasons, not because it belongs in the resource.